infra/base-images/base-builder/cargo

```diff
@@ -35,7 +35,11 @@ then
     # go into fuzz directory if not already the case
     cd fuzz || true
     fuzz_src_abspath=`pwd`
-    export RUSTFLAGS="$RUSTFLAGS --remap-path-prefix fuzz_targets=$fuzz_src_abspath/fuzz_targets"
+    # default directory is fuzz_targets, but some projects like image-rs use fuzzers
+    while read i; do
+        export RUSTFLAGS="$RUSTFLAGS --remap-path-prefix $i=$fuzz_src_abspath/$i"
+        # bash while syntax so that we modify RUSTFLAGS in main shell instead of a subshell
```

Comment:
Please begin with capital and end with period.

Reply:
done